frontend/admin/src/js/lang/fr.json

```diff
@@ -2788,5 +2788,45 @@
   "wHX/8C": {
     "defaultMessage": "Admin",
     "description": "Title tag for Admin site"
+  },
+  "+tzO5t": {
+    "defaultMessage": "Possède un diplôme"
+  },
+  "8lCjAM": {
+    "defaultMessage": "Priorité"
+  },
+  "Aaas0w": {
+    "defaultMessage": "L’heure de clôture sera automatiquement fixée à 23 h 59 dans le fuseau horaire du Pacifique.",
+    "description": "Helper message for changing the pool closing date"
+  },
+  "Gr3BwB": {
+    "defaultMessage": "Équité en matière d’emploi"
+  },
+  "Ude1JQ": {
+    "defaultMessage": "Texte",
+    "description": "Title for the application’s profile snapshot."
+  },
+  "hGlM9B": {
+    "defaultMessage": "Date de clôture (fuseau horaire du Pacifique)",
+    "description": "Closing Date field label for publish pool dialog in the Pacific time zone"
+  },
+  "j6V32h": {
+    "defaultMessage": "Date de clôture (fuseau horaire du Pacifique)",
+    "description": "Label for a pool advertisements expiry date in the Pacific time zone"
+  },
+  "m0JFE/": {
```

Comment:
Indeed, confusing because there are two descriptions that are identical:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/2cc2b2b2fc5f11e90ec755ace335f56bf01a27d5/frontend/admin/src/js/components/poolCandidate/ViewPoolCandidate/ViewPoolCandidatePage.tsx#L107-L121